specs/badge.md

```diff
@@ -0,0 +1,80 @@
+# fast-badge
+
+## Overview
+
+*Badge* component is used to highlight an item and attract attention or flag status, such as “New”, “Sale”, or other short phrases.
+
+### Use Cases
+Keep the *badge* label short and succinct &ndash; it should not be a sentence or a long phrase (recommend 2-3 words). Short phrases work well, such as “App of the day,” but keep in mind that they will generally become longer when localized. Keep the text to nouns and adjectives if possible (e.g. “New” or “New today”), so the user won’t think it is an actionable element.
+
+Typical use cases include, but are not limited to, denoting a sale or new item and flagging an item as part of a category
+  
+### Features
+
+A badge should allow the following properties:
+- `fill`, a string that passes the color value for the backplate. Defaults to "transparent".
+- `size`, the size the badge is rendered.
+
+### Risks and Challenges
+
+It's import to not over dominate the users screen with the use of *badge*. They should be used sparingly to denote special status or draw attention to a particular item.
+
+### Prior Art/Examples
+- [FAST-DNA (React)](https://explore.fast.design/components/badge)
+- [Material UI](https://material-ui.com/components/badges/)
+- [Lightning Design](https://www.lightningdesignsystem.com/components/badges/)
+- [Ant Design Badge](https://ant.design/components/badge/)
+- [Ant Design Lozenge](https://atlaskit.atlassian.com/packages/core/lozenge)
+- [Atlassian](https://atlaskit.atlassian.com/packages/core/badge)
+
+---
+
+### API
+
+*Component name:*
+- `fast-badge`
+
+*Attributes:*
+- `fill`: string
+- `size`: enum
+    - `small`
+    - `large`
+
+*Slots:*
+- `default`
+
+### Anatomy and Appearance
+
+*Template:*
+```
+<span style={{background: #FFD800}}>
```

Comment:
Is this still accurate based on the above comments regarding `fill`?

Reply:
Also, should we consider removing the span here? Relevant conversation [here](https://github.com/microsoft/fast-dna/pull/2743/files/984788cdf64fd400f3527dbb1c7e30c96b6b5524#r388507785)

Reply:
you mean changing too `<div part="root">` or just removing all together? Don't we need a containing element independent of text to style?

Reply:
Sorry - There's two things here:

1. Inline style doesn't seem necessary given the conversation above on fill.
2. Removing the span altogether as we can simply style the host, which can act as the containing element.